### PR TITLE
Added JSConf data to the data tracker #8

### PR DESCRIPTION
**Pull request by harshbisle**
This data is collected from the available organizational site https://jsconf.com/ and other resources online can change over time. /claim #8

Conference: JSConf 

Organizer: Local community or organization.  Each JSConf event is typically organized by a local team or community of JavaScript enthusiasts.

SEA: Since the organization or community is not fixed it's not feasible. Also, the agreements may vary depending on the specific event and its organizers.

Free Ticket: JSConf events do provide free or discounted tickets to speakers as a form of compensation or scholarship for their participation.

Reimburse Expenses: Partial, As the typical expenses that may be reimbursed can include travel costs, accommodation, meals, and other reasonable expenses.

Compensate Speakers: Often yes, But depends on the organizing community or organizations and the sponsors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ The conference dataset hosted in this repository tracks the following informatio
 | [ZIO World](https://zioworld.com) | Ziverge | Yes | Yes | No | Never |
 | [ScalarConf](https://www.scalar-conf.com) | Software Mill | No | Yes | No | Never | 
 | [PyCon](https://pycon.org) | PyCon Board Committee | Pending | No | Yes | Keynote |
+| [JSConf](https://jsconf.com/) | Local Community | No | Yes | Partial | Yes |
```
Comment (jdegoes) [inline README.md:33]: If whether to sign the SEA is up to the local chapter, then I would put `Varies` in the column rather than `No`.

Comment (harshbisle) [reply README.md:33]: As inferred from skimming through the resources very few would agree to sign the SEA, so I made it `No`. Do you want me to keep it as `varies`?

Comment (jdegoes) [reply README.md:33]: An alternate would be to ask each local community, but that seems like far worse than just saying the answer `Varies` (depending on the chapter).

Comment (harshbisle) [reply README.md:33]: I agree, JSConf is huge that it needs a separate spreadsheet of its own IMO.

Comment (jdegoes) [reply README.md:33]: All right, then change the column to `Varies`.

Comment (harshbisle) [reply README.md]: Done!

**Pull request merged**